README-dtq.md

```diff
@@ -0,0 +1,150 @@
+
+# DSpace
+
+[![Build](https://github.com/dataquest-dev/DSpace/actions/workflows/build.yml/badge.svg)](https://github.com/dataquest-dev/DSpace/actions/workflows/build.yml)
+[![codecov](https://codecov.io/gh/dataquest-dev/DSpace/branch/dtq-dev/graph/badge.svg?token=YI6CJNFI2H)](https://codecov.io/gh/dataquest-dev/DSpace)
+
+## Overview
```

Comment:
why do you duplicate readme contents?

Reply:
true, it shouldn't be here

Reply:
I copied whole content because for each point could be added additional information. I would like to keep the content from README.md.